com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -1498,6 +1498,12 @@ DepthOfFieldParameters PrepareDoFParameters(HDCamera camera)
             return parameters;
         }
 
+        static void GetDoResolutionScale(in DepthOfFieldParameters dofParameters, out float scale, out float resolutionScale)
```

Comment:
I assume Do**F**  here.

Also, does it need to be static?

Reply:
Yeah, I already fixed it :)